.github/workflows/tests.yml

```diff
@@ -294,9 +294,9 @@ jobs:
         uses: actions/cache@v3
         env:
           # Increase this value to reset cache
-          CACHE_NUMBER: 4
+          CACHE_NUMBER: 5
         with:
-          path: ~/conda_pkgs_dir
+          path: ~/.miniconda/pkgs
```

Comment:
But maybe I'm wrong: https://github.com/conda-incubator/setup-miniconda/blob/99a08d57b7564700c6a69d09e1ff94f7e6482e7d/src/outputs.ts#L35

Reply:
I get a ~/.conda/envs etc. for some commands, it is what conda does when it can't write the base installation (like how we set it up in Docker)

Reply:
https://github.com/conda-incubator/setup-miniconda#caching-packages

Reply:
Yikes. I guess it's possible to track down whether the github action sets the conda cache path with an environment variable, or to look up conda's exact cache path rules.

Reply:
Yeah, it was a trial to see if the cache was a problem for the macOS runners, the link I posted above sets the path it seems.